src/main/java/org/example/dto/MessageDTO.java

```diff
@@ -0,0 +1,18 @@
+package org.example.dto;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MessageDTO {
+    @JsonProperty("message")
+    private String message;
+
+    public String getMessage() {
+        return message;
+    }
+
+    public MessageDTO() {
```

Comment:
Constructors should be declared before other functions.
Do you need both of them?